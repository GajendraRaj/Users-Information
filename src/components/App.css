.App {
  text-align: center;
  font-family: sans-serif;
}

#paging-numbers {
  list-style: none;
  display: inline-flex;
}

#paging-numbers > li {
  margin-right: 0.4em;
  color: maroon;
  user-select: none;
  cursor: pointer;
  border: solid;
  border-color: grey;
}
