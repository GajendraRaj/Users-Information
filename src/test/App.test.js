import React from "react";
import { shallow } from "enzyme";
import App from "../components/App";
import Users from "../components/users";

describe("App component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<App />);
  });

  it("should render component correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should render title correctly", () => {
    expect(wrapper.find("h1").text()).toEqual("DB Users");
  });

  it("should render Users component", () => {
    expect(wrapper.find(Users).length).toEqual(1);
  });
});
