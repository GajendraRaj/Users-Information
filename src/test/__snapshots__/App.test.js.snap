// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render component correctly 1`] = `ShallowWrapper {}`;
